src/DotNetWorker.Grpc/GrpcStartupOptions.cs

```diff
@@ -15,5 +15,9 @@ internal class GrpcWorkerStartupOptions
         public string? RequestId { get; set; }
 
         public int GrpcMaxMessageLength { get; set; }
+
+        public string? ScriptRoot { get; set; }
```

Comment:
While I like this being short and not having `AzureWebJobs`... let's just make it `AzureWebJobsScriptRoot` so we don't have to do any custom binding later.